.github/workflows/UpdateCheckpoint_build_and_test_on_push.yml

```
@@ -0,0 +1,33 @@
+      env_var_name_7: HIQ_AZUREEVENTHUB_FULLYQUALIFIEDNAMESPACE
+      dotnet_version: 8.0.x
+    secrets:
+      badge_service_api_key: ${{ secrets.BADGE_SERVICE_API_KEY }}
+      test_feed_api_key: ${{ secrets.TASKS_TEST_FEED_API_KEY }}
+      env_var_value_1: ${{ secrets.EVENT_HUB_CONNECTION_STRING }}
+      env_var_value_2: ${{ secrets.HIQ_AZUREBLOBSTORAGE_APPID }}
+      env_var_value_3: ${{ secrets.HIQ_AZUREBLOBSTORAGE_CLIENTSECRET }}
+      env_var_value_4: ${{ secrets.HIQ_AZUREBLOBSTORAGE_CONNSTRING }}
+      env_var_value_5: ${{ secrets.HIQ_AZUREBLOBSTORAGE_TENANTID }}
+      env_var_value_6: ${{ secrets.HIQ_AZUREBLOBSTORAGE_TESTSORAGE01ACCESSKEY }}
+      env_var_value_7: ${{ secrets.HIQ_AZUREEVENTHUB_FULLYQUALIFIEDNAMESPACE }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)